scripts/build-release.sh

```diff
@@ -222,7 +222,7 @@ ROCPROFSYS_GENERAL_ARGS="-DROCPROFSYS_CPACK_SYSTEM_NAME=${DISTRO} -DROCPROFSYS_R
 ROCPROFSYS_BUILD_ARGS="-DROCPROFSYS_BUILD_TESTING=OFF -DROCPROFSYS_BUILD_EXAMPLES=OFF -DROCPROFSYS_BUILD_PAPI=ON -DROCPROFSYS_BUILD_LTO=${LTO} -DROCPROFSYS_BUILD_HIDDEN_VISIBILITY=${HIDDEN_VIZ} -DROCPROFSYS_BUILD_STATIC_LIBGCC=${LIBGCC} -DROCPROFSYS_BUILD_STATIC_LIBSTDCXX=${LIBSTDCXX} -DROCPROFSYS_BUILD_RELEASE=ON"
 ROCPROFSYS_USE_ARGS="-DROCPROFSYS_USE_MPI_HEADERS=ON -DROCPROFSYS_USE_OMPT=ON -DROCPROFSYS_USE_PAPI=ON"
 TIMEMORY_ARGS="-DTIMEMORY_USE_LIBUNWIND=ON -DTIMEMORY_BUILD_LIBUNWIND=ON -DTIMEMORY_BUILD_PORTABLE=ON"
-DYNINST_ARGS="-DROCPROFSYS_BUILD_DYNINST=ON -DDYNINST_USE_OpenMP=ON $(echo -DDYNINST_BUILD_{TBB,BOOST,ELFUTILS,LIBIBERTY}=ON) -DDYNINST_BOOST_DOWNLOAD_VERSION=${BOOST_VERSION}"
+DYNINST_ARGS="-DROCPROFSYS_BUILD_DYNINST=ON -DROCPROFSYS_USE_OpenMP=ON $(echo -DROCPROFSYS_BUILD_{TBB,BOOST,ELFUTILS,LIBIBERTY}=ON) -DROCPROFSYS_BOOST_DOWNLOAD_VERSION=${BOOST_VERSION}"
```

Comment:
```suggestion
DYNINST_ARGS="-DROCPROFSYS_BUILD_DYNINST=ON -DROCPROFSYS_USE_OpenMP=ON $(echo -DROCPROFSYS_BUILD_{TBB,BOOST,ELFUTILS,LIBIBERTY}=ON) -DROCPROFSYS_BOOST_DOWNLOAD_VERSION=${BOOST_VERSION}"
```
The CMake module is just using `BOOST_DOWNLOAD_VERSION`. See previous comments/suggestions which adds the "ROCPROFSYS" prefix.

Reply:
It doesn't look like DYNINST_USE_OpenMP is being passed down either. Might as well just remove it from this script.

Reply:
Should I change to -DDYNINST_USE_OpenMP=ON as there is a USE_OpenMP variable in dyninst.

Reply:
I think we can just remove. It was never used before and I don't want it to be confused for an option that removed "OpenMP" support for rocprof-sys as a whole.

Reply:
Ok I removed it.